app/controllers/miq_policy_controller/policies.rb

```diff
@@ -182,7 +182,7 @@ def miq_policy_edit_events
   end
 
   def miq_policy_edit_conditions
-    assert_privileges('miq_policy_edit_conditions')
+    assert_privileges('miq_policy_conditions_assignment')
```

Comment:
Before this PR, you couldn't add events or conditions to the profile, like seen below:
![image](https://user-images.githubusercontent.com/19339/119040392-5b62fb80-b983-11eb-9e25-21046c948366.png)